HW_3/task_1.py

```diff
@@ -0,0 +1,18 @@
+"""
+Задайте список целых чисел. Найдите сумму элементов списка, имеющих нечетные индексы.
+
+Ввод: значение типа <list> (либо значение типа <int> – размерность списка)
+Вывод: значение типа <int>
+
+Примеры:
+[2, 3, 5, 9, 3]
+12
+
+[5, 1, 5, 2, 7, 11]
+14
+"""
+import random
+new_list = [random.randint(1, 99)
+            for r in range(int(input('Введите размер списка: ')))]
+print('Сумма чисел с нечетными индексами в списке ', new_list, 'равна',
+      sum(new_list[i] for i in range(len(new_list)) if i % 2 != 0))
```

Comment:
Можно проще.
```suggestion
      sum(new_list[i] for i in range(1, len(new_list), 2))
```
Или еще проще.
```suggestion
      sum(new_list[1::2])
```